packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -5534,6 +5561,7 @@ export type StyleQueue = {
   rules: Array<Chunk | PrecomputedChunk>,
   hrefs: Array<Chunk | PrecomputedChunk>,
   sheets: Map<string, StylesheetResource>,
+  nonce?: ?Chunk,
```

Comment:
let's make the property required. fewer hidden class lookups when the nonce is added later